dask_ml/model_selection/_incremental.py

```diff
@@ -205,9 +216,15 @@ def get_futures(partial_fit_calls):
     start_time = time()
 
     # async for future, result in seq:
-    while True:
+    for _i in itertools.count():
         metas = yield client.gather(new_scores)
 
+        if _i % int(verbose) == 0:
+            idx = np.argmax([m["score"] for m in metas])
```

Comment:
Can `metas` ever be empty here, causing a ValueError from taking the max of an empty sequence?

Reply:
No, `metas` could never be empty.

Why? Values gets added to `new_scores` for every non-zero value of `instructions: Dict[str, int]`. If `sum(instructions.values()) == 0` the loop is broken.